libs/geometry/src/TangentSpaceMesh.cpp

```diff
@@ -321,6 +323,252 @@ void flatShadingMethod(const TangentSpaceMeshInput* input, TangentSpaceMeshOutpu
     output->triangleCount = outTriangleCount;
 }
 
+class MikktspaceImpl {
```

Comment:
maybe MikktspaceImpl could be in its own cpp?

Reply:
done